packages/world/src/schemas/RouteAccess.sol

```diff
@@ -0,0 +1,86 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0;
+
+import { console } from "forge-std/console.sol";
+import { IStore } from "store/IStore.sol";
+import { StoreSwitch } from "store/StoreSwitch.sol";
+import { StoreCore } from "store/StoreCore.sol";
+import { SchemaType } from "store/Types.sol";
+import { Bytes } from "store/Bytes.sol";
+import { Schema, SchemaLib } from "store/Schema.sol";
+import { PackedCounter, PackedCounterLib } from "store/PackedCounter.sol";
+import { SliceLib } from "store/Slice.sol";
+
+// -- User defined schema and tableId --
+struct RouteAccessSchema {
+  bool value;
+}
+
+// -- Autogenerated library to interact with tables with this schema --
+// TODO: autogenerate
+
+library RouteAccessSchemaLib {
+  /** Get the table's schema */
+  function getSchema() internal pure returns (Schema schema) {
+    schema = SchemaLib.encode(SchemaType.BOOL);
+  }
+
+  /** Register the table's schema */
+  function registerSchema(uint256 tableId) internal {
+    StoreSwitch.registerSchema(tableId, getSchema());
+  }
+
+  function registerSchema(uint256 tableId, IStore store) internal {
+    store.registerSchema(tableId, getSchema());
+  }
+
+  /** Set the table's data */
+  function set(
+    uint256 tableId,
```

Comment:
agree, would slightly prefer `tableId` i think, but so far we mostly used `table` in `store`. To avoid merge conflicts with @dk1a's store PRs I wouldn't change that in this PR. Also I expect the files inside `schemas` and `tables` to be replaced by auto-generated ones once that's ready